pinot-core/src/main/java/org/apache/pinot/core/data/manager/SegmentDataManager.java

```diff
@@ -62,6 +62,14 @@ public synchronized boolean decreaseReferenceCount() {
     }
   }
 
+  /**
+   * Indicates whether this segment is physically in memory or downloaded to disc.
+   * @return
+   */
+  public boolean hasLocalData() {
```

Comment:
`false` indicates the segment is downloaded to disc but not loaded into memory?

Reply:
`false` indicates the segment is not downloaded to local disc

Reply:
I see. Can you please update the javadoc? The current one is misleading